test/python/terraform-functions/__snapshots__/test.ts.snap

```diff
@@ -0,0 +1,48 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`python terraform functions test synth synth generates JSON 1`] = `
+"{
+  \\"//\\": {
+    \\"metadata\\": {
+      \\"version\\": \\"stubbed\\",
+      \\"stackName\\": \\"python-simple\\",
+      \\"backend\\": \\"local\\"
+    }
+  },
+  \\"terraform\\": {
+    \\"required_providers\\": {
+      \\"aws\\": {
+        \\"version\\": \\"~> 2.0\\",
+        \\"source\\": \\"aws\\"
+      }
+    }
+  },
+  \\"provider\\": {
+    \\"aws\\": [
+      {
+        \\"region\\": \\"eu-central-1\\"
+      }
+    ]
+  },
+  \\"module\\": {
+    \\"CustomVpc\\": {
+      \\"name\\": \\"custom-vpc\\",
+      \\"source\\": \\"terraform-aws-modules/vpc/aws\\",
+      \\"version\\": \\"2.77.0\\",
+      \\"//\\": {
+        \\"metadata\\": {
+          \\"path\\": \\"python-simple/CustomVpc\\",
+          \\"uniqueId\\": \\"CustomVpc\\"
+        }
+      }
+    }
+  },
+  \\"output\\": {
+    \\"computed\\": {
+      \\"value\\": {
+        \\"jsii_byref\\": \\"Object@10006\\"
```

Comment:
This doesn't seem to have resolved correctly.

Reply:
Yeah, I think it was due to a wrong return type of the functions. The current version has a different problem, so one step at a time I guess :D